extendedcopy.go

```diff
@@ -236,6 +239,9 @@ func (opts *ExtendedCopyGraphOptions) FilterAnnotation(key string, regex *regexp
 // when using both FilterArtifactType and FilterAnnotation, it's recommended to call
 // FilterArtifactType first.
 func (opts *ExtendedCopyGraphOptions) FilterArtifactType(regex *regexp.Regexp) {
+	if regex == nil {
```

Comment:
In the case of `regex == nil`, why does `FilterAnnotation` proceeds filtering, while `FilterArtifactType` just returns ? Should they have the same behavior?